doc/api/fs.markdown

```diff
@@ -657,13 +659,13 @@ that leaves you vulnerable to race conditions: another process may remove the
 file between the calls to `fs.exists()` and `fs.open()`.  Just open the file
 and handle the error when it's not there.
 
-`fs.exists()` is **deprecated**.
+
 
 ## fs.existsSync(path)
 
-Synchronous version of `fs.exists`.
+Synchronous version of [`fs.exists`](fs.html#fs_fs_exists_path_callback).
 
-`fs.existsSync()` is **deprecated**.
+`fs.existsSync()` is **deprecated**. For future proof alternatives please check out [`fs.statSync`](fs.html#fs_fs_statsync_path) or [`fs.access`](fs.html#fs_fs_accesssync_path_mode).
```

Comment:
It should be \`fs.accessSync\` instead of \`fs.access\`. The link URL is correct though.